ocpmodels/trainers/tune_hpo_trainer.py

```diff
@@ -0,0 +1,54 @@
+import copy
+import datetime
+import json
+import os
+import sys
+
+import ray
+import torch
+from ocpmodels.common.meter import Meter, mae, mae_ratio
+from ocpmodels.common.registry import registry
+from ocpmodels.modules.normalizer import Normalizer
+from ocpmodels.trainers import BaseTrainer
+from ray import tune
+
+
+@registry.register_trainer("tune_hpo")
+# BaseTrainer
+class TuneHPOTrainer(tune.Trainable):
+    # TODO(Brandon): make _setup general to any trainer, once config dicts are standard
+    def _setup(self, config):
+        self.trainer = registry.get_trainer_class("simple")(
+            task=config["task"],
+            model=config["model"],
+            dataset=config["dataset"],
+            optimizer=config["optim"],
+            identifier="",
+            is_debug=True,
+            seed=0,
+            logger=None,
+        )
+
+        print("Device = {dev}".format(dev=self.trainer.device))
+        # load() is part of the simple trainer but will need to added in the future
+        # self.trainer.load()
```

Comment:
Not sure I completely follow. Where does `.load` for `SimpleTrainer` get called?

Reply:
It is actually called in simple_trainer see this [line](https://github.com/Open-Catalyst-Project/baselines/blob/master/ocpmodels/trainers/simple_trainer.py#L67). I think this should be changed but I was planning to change in a different branch aimed at standardizing the trainer input

Reply:
Ah, cool, makes sense.